### PR TITLE
Be more aggressive on avx512

### PR DESCRIPTION
**Pull request by fenrus75**

### CODE REVIEW

File: autospec/pkg_integrity.py [hunk 1]
```diff
@@ -665,7 +665,7 @@ def get_verifier(filename):
 def get_input(message, default):
     """Parse user input."""
     try:
-        import_key = input(message)
+        import_key = 'y'
```
Comment (phmccarty) [inline autospec/pkg_integrity.py:668]: I don't think we should auto-accept new pubkey imports...

File: autospec/specfiles.py [hunk 2]
```diff
@@ -671,7 +671,7 @@ def write_license_files(self):
                 file2 = self.hashes[lfile]
                 # Use the absolute path to the source license file b/c we don't know for sure where we are
                 lfile = lfile.replace(self.version, "%{version}")
-                self._write_strip("cp " + "%{_builddir}/" + lfile + " %{buildroot}/usr/share/package-licenses/" + self.name + "/" + file2 + "\n")
+                self._write_strip("cp " + "%{_builddir}/" + lfile + " %{buildroot}/usr/share/package-licenses/" + self.name + "/" + file2 + " || :\n")
```
Comment (phmccarty) [inline autospec/specfiles.py:674]: What is the reason for ignoring copy errors?

Comment (fenrus75) [reply autospec/specfiles.py:674]: there's a few packages where the license file is a dead symlink which has no other way for us to work around

Comment (phmccarty) [reply autospec/specfiles.py:674]: We appear to skip the license file for that case... (see https://github.com/clearlinux/autospec/pull/565) I just tested running autospec on a tarball with a broken license symlink, and can confirm that the exception is handled as expected.

Do you remember which packages are affected? I can test those...

**Pull request merged**